radix-engine-interface/src/blueprints/resource/access_rules.rs

```diff
@@ -191,13 +191,13 @@ impl AuthorityRules {
         );
     }
 
-    pub fn set_owner_rule(&mut self, rule: AccessRule, mutability: AccessRule) {
+    pub fn set_owner_authority(&mut self, rule: AccessRule, mutability: AccessRule) {
```

Comment:
The consensus in the thread I think was that we specify in full - so eg:
```
.set_owner(AuthorityRules {
   may_perform: rule!(...),
   may_change: rule!(..),
})
```

What do you think? :)

Reply:
Although TBH I'd suggest: `set_owner(rule: AccessRule)` and set owner to mutate itself?

And have a separate `set_owner_mutability` for the rare case it wants to be fixed?

Reply:
I'll keep it as is for now and wait for full consensus :)